manifests/plugin/ansible.pp

```diff
@@ -54,16 +51,19 @@
   $foreman_ssl_ca = pick($foreman_proxy::foreman_ssl_ca, $foreman_proxy::ssl_ca)
 
   file { "${foreman_proxy::config_dir}/ansible.cfg":
+    ensure  => absent,
+  }
+  ~> file { "${foreman_proxy::dir}/.ansible.cfg":
```

Comment:
This file is shipped in the RPM:
https://github.com/theforeman/foreman-packaging/blob/3f3cc32ef6ab3f4f7107b26972daf2189d976303/packages/plugins/rubygem-smart_proxy_ansible/rubygem-smart_proxy_ansible.spec#L139
That causes problems with idempotency.

Reply:
What would be the right course of action then? Stop managing it, effectively making it become a dangling symlink and then eventually dropping that symlink from packaging?

Reply:
I think both files are part of packaging, so stopping managing them sounds OK to me?

```console
$ rpm -ql rubygem-smart_proxy_ansible-3.3.1-2.el8sat.noarch |grep ansible.cfg
/etc/foreman-proxy/ansible.cfg
/usr/share/foreman-proxy/.ansible.cfg
```